.circleci/generate_circleci_config.py

```diff
@@ -9,9 +9,9 @@
 from jinja2 import Environment, FileSystemLoader
 
 IMAGE_MAP = collections.OrderedDict([
-    ("1.10.5", ["alpine3.10", "buster", "rhel7"]),
-    ("1.10.6", ["alpine3.10", "buster"]),
-    ("1.10.7", ["alpine3.10", "buster"]),
+    ("1.10.5-6", ["alpine3.10", "buster", "rhel7"]),
```

Comment:
Hmmm hmmm hmmm.

I wonder if we could extract this info out of the image, just so we don't have to (remember, and fail) to update this info in two places.

Reply:
Other than the fact that we don't currently set the right version:


```
$ docker image inspect astronomerinc/ap-airflow:1.10.7-alpine3.10-onbuild-4208 -f '{{index .Config.Labels "io.astronomer.docker.airflow.version"}}'
1.10.7
```

Reply:
Yeah, I think it has been there since the start and I feel the intention was to show **Airflow Version**:

https://github.com/astronomer/ap-airflow/blob/10167d89afd233a9633496456d4c240e1da2d5d5/1.10.7/alpine3.10/Dockerfile#L23

https://github.com/astronomer/ap-airflow/blob/10167d89afd233a9633496456d4c240e1da2d5d5/1.10.7/alpine3.10/Dockerfile#L26

Reply:
Based on our discussion added a new label to specify AC Version

Reply:
Based on that commit, we will need to change the VERSION only in `generate_circleci_config.py` and it will change the VERSIONS in all Dockerfiles